lang/en/tool_mergeusers.php

```diff
@@ -194,3 +194,9 @@
 $string['finishtime'] = 'Finished merging at {$a}';
 $string['timetaken'] = 'Merge took {$a} seconds';
 $string['privacy:metadata'] = 'The Merge User Accounts plugin does not store any personal data.';
+$string['queue_to_process_pending_merge_requests'] = 'Get queue of the pending merge requests and launch adhoc task for each single request';
+$string['cannotfinduser'] = 'Cannot find the user with {$a->userfield} {$a->uservalue}. Change the input parameters.';
+$string['toomanyusers'] = 'ERROR: More than one user found with {$a->userfield} {$a->uservalue}.';
```

Comment:
I would unify the kind of sentence between `cannotfinduser` and `toomanyusers`. I think we should remove the text `ERROR: ` from the `toomanyusers`. Imagine reading the tracelog of the exception and in addition there the text `ERROR: `. There it is cristal clear that it is an error, hehehe. So this text is unnecessary.

Reply:
removed "ERROR: ..." string text.

Reply:
done